experimental/auth-logic/unix_epoch_time_wrapper.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package authlogic
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// This file contains a wrapper that produces the current
```

Comment:
Could this be moved up and used as the doc comment for the package? 
Although this package name is present in multiple files, and they have different documentation.

Reply:
Based on my new discovery, the convention is to move this comment right after the "package" statement. Also, the doc comment for the package can be removed.